code/game/machinery/vending.dm

```diff
@@ -3140,6 +3140,13 @@ var/global/num_vending_terminals = 1
 		/obj/item/weapon/fakeposter_kit = 1,
 		/obj/structure/closet/crate/flatpack/ancient/condiment_dispenser = 1,
 		/obj/structure/closet/crate/flatpack/ancient/chemmaster_electrolyzer = 1,
+		/obj/item/weapon/storage/box/mysterycubes = 2,
+		/obj/item/weapon/storage/box/mystery_vial = 5,
+		/obj/item/weapon/storage/box/mystery_circuit = 1,
+		/obj/item/weapon/storage/box/large/mystery_material = 5,
+		/obj/item/weapon/storage/box/large/mystery_material/odd = 5,
+		/obj/structure/closet/crate/freezer/bootlegpicnic = 3,
+		/obj/structure/vendomatpack/trader = 1,
```

Comment:
```suggestion
		/obj/structure/vendomatpack/trader = 1
```
at the end of a list, you do not put a comma

Reply:
or maybe I'm wrong... case by case maybe? No clue, sorry!

Reply:
In most languages the trailing comma would be a problem, but BYOND doesn't give a shit, probably to make this specific formatting easier
It's ugly but doesn't cause any problems AFAIK

Reply:
I used to leave them out but most times I see a list likely to be touched by a few people they leave a trailing comma. I always assumed it was to prevent the next person from accidentally forgetting it.

Reply:
mark as resolved